geolucidate/tests/tests.py

```diff
@@ -87,7 +91,7 @@ def test_parser():
         (u"70 ° 57N 070 ° 05W", ['N', '70', '57', '00', 'W', '070', '05', '00']),
         (u"""(45º10'17"N 076º23'46"W)""", ['N', '45', '10', '17', 'W', '076', '23', '46']),
         # Note that the degree and minute punctuation are actually backwards; we support it anyway.
-        (u"""(45º10"17'N 076º23"46'W" """, ['N', '45', '10', '17', 'W', '076', '23', '46']),
+        # (u"""(45º10"17'N 076º23"46'W")""", ['N', '45', '10', '17', 'W', '076', '23', '46']),
```

Comment:
@AvenzaOleg this one was missed when you created the new tests. I'm thinking we don't support this format?

the minute/second marks are switched

Reply:
It does not look like a valid format, let's keep it commented